static/bidder-info/boldwin.yaml

```diff
@@ -1,6 +1,6 @@
 endpoint: "http://ssp.videowalldirect.com/pserver"
 maintainer:
-  email: "wls_demo_box@smartyads.com"
+  email: "info@bold-win.com"
```

Comment:
Sent email for verification.

Reply:
We haven't received a reply yet. Please resend. @Wls-demo

Reply:
@gargcreation1992 we have replied, please check

Reply:
received. thanks!